README.md

```diff
@@ -329,13 +330,70 @@ If you have classes with a lot of columns and you filter them often with the sam
           {
             "name": "email",
             "filterSortToTop": true
-          }          
+          }
         ]
       }
     }
 ]
 ```
 
+### Custom Cloud Code scripts
+
+You can specify custom cloud code scripts with the `scripts` option:
```

Comment:
I'd name this feature more general just "Scripts". Maybe in the future we add a parameter `url` so that instead of a Cloud Function a RESTful API can be invoked with the Parse Object as payload.

```suggestion
### Scripts

You can specify scripts to execute Cloud Functions with the `scripts` option:
```